Examples/Examples/FormExampleView.swift

```diff
@@ -44,22 +44,25 @@ struct FormExampleView: View {
                 
                 // Present a FormView in a native SwiftUI sheet
                 .sheet(isPresented: $isPresented) {
-                    if #available(iOS 16.4, *) {
-                        FormView()
-                            .padding()
-                            .presentationBackground(.thinMaterial)
-                            .presentationBackgroundInteraction(.enabled(upThrough: .medium))
-                            .presentationDetents([.medium])
-                    } else {
-                        FormView()
-                            .padding()
-                    }
-                    #if targetEnvironment(macCatalyst)
-                    Button("Dismiss") {
-                        isPresented = false
+                    Group {
```

Comment:
The `.environmentObject(formViewModel)` is moved into the sheet, otherwise it is not made available in the hierarchy in Mac Catalyst. Opening the form resulted in a crash.